JobWorkers/JobWorkerManualRespond.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace JobWorkers;
+
+/**
+ * KPHP job workers — real parallelism: separate processes to parallelize large (unforkable) code pieces and CPU work.
+ *
+ * "Manual" job requests are more compilcated then @see JobWorkerSimple. They fork as follows:
+ * * takes a request
+ * * prepares data to respond
+ * * sends a response, unfreezing http worker
+ * * keeps working in the background, along with the http worker
+ * * finishes
+ * On wait(), http worker continues in the middle of job worker execution. After wait(), they work simultaneously.
```

Comment:
I think it's worth noting that http worker is not blocking on `wait`, but asynchronously runs another resumable / coroutine